### PR TITLE
Modified Invalueset for string overload.

### PR DESCRIPTION
**Pull request by jbradl11**
Pull requests into cql-execution require the following.
Submitter and reviewer should ✔ when done.
For items that are not-applicable, mark "N/A" and ✔.

This PR, updates the "InValueSet" string overload to throw an error if the string "code" exists in the value set but in multiple code systems.

From the Spec:
`For the String overload, if the given valueset contains a code with an equivalent code element, the result is true. Note that for this overload, because the code being tested cannot specify code system information, if the resolved value set contains codes from multiple code systems, a run-time error is thrown because the operation is ambiguous.`

[CDS Connect](https://cds.ahrq.gov/cdsconnect) and [Bonnie](https://github.com/projecttacoma/bonnie) are the main users of this repository. 
It is strongly recommended to include a person from each of those projects as a reviewer.

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Code diff has been done and been reviewed (it does not contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] ~Code coverage has not gone down and all code touched or added is covered.~
- [x] ~All dependent libraries are appropriately updated or have a corresponding PR related to this change~

**Reviewer:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: src/datatypes/clinical.coffee [hunk 1]
```diff
@@ -30,7 +30,22 @@ module.exports.ValueSet = class ValueSet
...
-    codesInList(toCodeList(code), @codes)
+    codesList = toCodeList(code)
+    # InValueSet String Overload
+    if codesList.length == 1 and typeof codesList[0] is 'string'
+      codes_codeSystem = {}
+      for c2 in @codes
+        if codesList[0] == c2.code
+          # Check if code exists but with different codesystem,
+          if codes_codeSystem[c2.code]? and codes_codeSystem[c2.code] != c2.system
+            throw new Error('Duplicate codes in different code systems in same valueset')
```
Comment (losborne) [inline src/datatypes/clinical.coffee]: I suggest changing the wording of this error to use the wording from the CQL spec.  Something like:
`In (valueset) is ambiguous because resolved value set contains codes from multiple code systems.`

We could also put the code in the error message like this, which might be helpful:
`In (valueset) is ambiguous--multiple matches for #{code} found in value set with different code systems.`
